apps/ios/SimpleXChat/APITypes.swift

```diff
@@ -1878,13 +1878,17 @@ public enum XFTPErrorType: Decodable {
     case NO_FILE
     case HAS_FILE
     case FILE_IO
+    case TIMEOUT
+    case REDIRECT(redirectError: String)
     case INTERNAL
 }
 
 public enum RCErrorType: Decodable {
     case `internal`(internalErr: String)
     case identity
     case noLocalAddress
+    case newController
+    case notDiscovered
     case tlsStartFailed
```

Comment:
`tLSStartFailed` ?

Reply:
unless it's renamed in backend from RCETLSStartFailed to RCETlsStartFailed (which is more work)